app/views/scheduler/_modals.html.slim

```diff
@@ -57,4 +60,32 @@
             span  A Thirsty Thursday
           .modal-footer
             button.btn.btn-default data-dismiss="modal" type="button"  Close
-            button.btn.btn-primary#generate-with-options type="button"  Generate
\ No newline at end of file
+            button.btn.btn-primary#generate-with-options type="button"  Generate
+#how-to-modal.modal.fade aria-hidden="true" aria-labelledby="course-title" role="dialog" tabindex="-1"
```

Comment:
agreed. Though I'm thinking since there's so much info, turning it into an accordion